x11-wm/cwm/cwm-6.6.ebuild

```diff
@@ -1,35 +1,34 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit eutils toolchain-funcs
+inherit desktop eutils toolchain-funcs
```

Comment:
I don't see eutils being used in the ebuild

Reply:
I've removed it from the ebuild, thanks again